cfme/fixtures/pytest_selenium.py

```diff
@@ -858,9 +858,13 @@ def first_from(*locs, **kwargs):
 
     """
     assert len(locs) > 0, "You must provide at least one locator to look for!"
+    single = kwargs.pop("single", True)
     for locator in locs:
         try:
-            return element(locator, **kwargs)
+            if single:
```

Comment:
This function is "first_from". Based on the docs, I would expect this to return an element (the first), but the docs aren't updated so the new behavior is hidden and magical. If we ever decide to have elements use a 'single' kwarg, it will be surprisingly intercepted by this function (kwargs popping is something we do a lot, but it's become a crutch that ought to be avoided). Having the return type change from a single thing to an array of things can be confusing, as evidenced by out having a separate 'element' and an 'elements' function.

Much like element/elements, I think this behavior ought to split out into its own function that always returns a list of elements (`first_n_from(n, *locs, **kwargs)`, or something...?), and `first_from` can be adapted to use it (`return first_n_from(1, *locs, **kwargs)[0]`